core-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -274,6 +275,25 @@ public void run() {
   }
 
   /**
+   * Returns a path in the form of <package>/<class_name>.
+   *
+   * <p>This is so that the intermediate map in JavaUtil.getFileFromCloudPath() method recognises
+   * the path key.
+   *
+   * <p>https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/338
+   */
+  @VisibleForTesting
+  protected String cropCloudPath(String originalCloudPath) {
+    int start = originalCloudPath.indexOf("main/java");
```

Comment:
JavaUtil: package/class

ninja/joao/guestbook/SignGuestbookServlet.java

originalCloudPath:

jamjavatest/guestbook/src/main/java/ninja/joao/guestbook/SignGuestbookServlet.java

Looking around ProjectFileIndex, I couldn't find any method that would give us anything different and more useful than getPackage() and getClass() joined.